legal-api/src/legal_api/models/batch_processing.py

```diff
@@ -22,6 +23,11 @@
 from .db import db
 
 
+def default_trigger_date():
+    """Set default value for trigger_date."""
+    return datetime.utcnow() + datedelta(days=42)
```

Comment:
Don't think we should have a default trigger date.

The batch related tables may be used for other batch related processes in the future.